### DISCUSSION TITLE
after Create, how to redirect 'back'

### DISCUSSION THREAD

**Discussion by djamesfar** [filamentphp/filament#12410] [Help] [answered]
### Package

Form builder

### Package Version

v3.2.63

### How can we help you?

Users set filters in the Table, then create new record.  How to redirect users back to existing index (like clicking browser back) so that their filters are preserved instead of using the index route?  The redirectUrl expects a route, but how do I simply go back to previous page?

**Comment by leandrocfe** [comment]
Did you try `back()`?
https://laravel.com/docs/11.x/helpers#method-back

**Comment by djamesfar** [comment, marked as answer]
I found this, using previousUrl:
```
protected function getRedirectUrl(): string
    {
        return $this->previousUrl ?? $this->getResource()::getUrl('index');
    }
```
